### PR TITLE
Fethas Carnival of Horrors:Fethas Please Stop

### PR DESCRIPTION
**Pull request by Aurorablade**
adds organic debris field event inspired by https://github.com/d3athrow/vgstation13/pull/8169
and copy pasted from meaty ores

....Adds MeteorOps

I need to add it to the event stuff but forget how, i need to lessen the damage (point me in the right direction here)...and make the 'meteors' spin....

:cl: Fethas
rscadd:add Organic debris field event
rscadd:Adds MeteorOps
/:cl:

### CODE REVIEW

File: code/modules/events/meaty_gore.dm [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+/obj/effect/space_dust/gore/Bump(atom/A)
+	if(prob(20))
+		spawn(1)
+			for(var/mob/M in range(10, src))
+				if(!M.stat && !istype(M, /mob/living/silicon/ai))
+					shake_camera(M, 3, 1)
+	if (A)
+		playsound(src.loc, 'sound/effects/meteorimpact.ogg', 40, 1)
+		walk(src,0)
+		invisibility = 101
```
Comment (tigercat2000) [inline code/modules/events/meaty_gore.dm]: what the fuck

File: code/modules/events/meaty_ops.dm [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/datum/event/dust/goreop/announce()
+	var/meteor_declaration = "MeteorOps have declared thier intent to utterly destroy [station_name()] with thier own bodies, and dares the crew to try and stop them."
+	command_announcement.Announce(meteor_declaration, "Declaration of 'War'", 'sound/effects/siren.ogg')
+
+/datum/event/dust/goreop/setup()
+	qnty = rand(45,125)
+
+/datum/event/dust/goreop/start()
+	while(qnty-- > 0)
```
Comment (tigercat2000) [inline code/modules/events/meaty_ops.dm]: this while loop hurts my soul

File: code/modules/events/meaty_gore.dm [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/datum/event/dust/gore/announce()
+		command_announcement.Announce("Unknown biological debris have been detected near [station_name()], please stand-by.", "Debris Alert")
+
+/datum/event/dust/gore/setup()
+	qnty = rand(45,125)
+
+/datum/event/dust/gore/start()
+	while(qnty-- > 0)
```
Comment (tigercat2000) [inline code/modules/events/meaty_gore.dm]: :eyes:

**Comment by lolzyking** [comment]
what the fuck is meteor ops

File: code/modules/events/meaty_ops.dm [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+		new /obj/effect/space_dust/gore()
+		if(prob(10))
+			sleep(rand(10,15))
```
Comment (Fox-McCloud) [inline code/modules/events/meaty_ops.dm]: ummmm, this seems dangerous.

**Comment by Keilden** [comment]
More random events is always a good thing. :+1:

File: code/modules/events/meaty_ops.dm [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+		spawn(1)
+			for(var/mob/M in range(10, src))
+				if(!M.stat && !istype(M, /mob/living/silicon/ai))
+					shake_camera(M, 3, 1)
+	if (A)
+		playsound(src.loc, 'sound/effects/meteorimpact.ogg', 40, 1)
+		walk(src,0)
+		new /obj/effect/decal/cleanable/blood(get_turf(A))
+		if(ismob(A))
+			A.ex_act(strength)
```
Comment (Crazylemon64) [inline code/modules/events/meaty_ops.dm]: So these will `ex_act(1)` any poor soul unfortunate enough to get hit by them?

**Comment by Aurorablade** [comment]
reminder:I litterly copied this from Meaty Ores Event...so you are gonna have to tell me what i need to change where....

**Comment by Aurorablade** [comment]
hold that thought....I am gonna look on TG as we use thier system.....

**Comment by Fox-McCloud** [comment]
@Aurorablade 

No, we don't, actually. We use um....Bay's event system.

**Comment by Regen1** [comment]
I am not so sure about this.. 
As @necaladun has already said, we already have enough "memes" as is, and the whole "meaty ores" is already a thing

**Comment by Aurorablade** [comment]
Admin only atm and..i never do anything with an intent to be a 'meme' i do it with an intent of fun..but if people for some reason feel its too much or a meme i will close the PR....

**Comment by Regen1** [comment]
Oh it's admin only? Then that's fine I suppose, as long as it is not spammed

**Pull request merged**